### PR TITLE
Add warnings for some deprecated tools

### PR DESCRIPTION
**Pull request by Mimickal**
- Made the page for tool_pro be more explicit about why it shouldn't be used.
- Added a page for HEK+ with a chunk of info on why it shouldn't be used.
- Added "deprecated tools" list to the tool root page.
- Modified workflows so those pages suggest alternatives.

### CODE REVIEW

File: src/content/h1/tools/hek-plus/readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+**Halo Editing Kit Plus (HEK+)** is a tool for extracting tags from Halo Custom
+Edition maps, as well as "protecting" maps to prevent tag extraction.
```
Comment (csauve) [inline src/content/h1/tools/hek-plus/readme.md:2]: The map cache file page has a section about protected maps and why they’re bad that can be linked to from here.

File: src/content/h1/tools/hek-plus/readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+```.alert danger
+**HEK+ should not be used!** It introduces a number of issues to the tags it
+extracts.
```
Comment (csauve) [inline src/content/h1/tools/hek-plus/readme.md:6]: Soft wrapping would be preferable for me over hard wrapping at 80 chars, makes paragraphs a bit easier to edit later without worrying about lines.

**Review by csauve — APPROVED** [review]

File: src/content/h1/tools/hek-plus/readme.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+**Halo Editing Kit Plus (HEK+)** is a tool for extracting tags from Halo Custom
```
Comment (csauve) [inline src/content/h1/tools/hek-plus/readme.md:1]: Thoughts on using past tense for deprecated tools? “HEK+ was...”

Comment (Mimickal) [reply src/content/h1/tools/hek-plus/readme.md:1]: I think past tense is a bit strange from a wiki perspective. These tools are still out there and can be used, it's just that their usage is discouraged. I'd prefer to reserve past tense for things that are no longer available at all.

For a silly comparison, the Wikipedia page for the Byzantine Empire uses "was", while the Wiki page for E.T. on the Atari 2600 uses "is".

**Pull request merged**